src/app/portlets/dot-edit-page/content/services/html/iframe-edit-mode.js.ts

```diff
@@ -90,9 +90,6 @@ export const EDIT_PAGE_JS = `
             });
         }
     })
-
-    window.${MODEL_VAR_NAME}.next(getDotNgModel());
-    window.getDotNgModel = getDotNgModel;
```

Comment:
this was not removed in PR --> https://github.com/dotCMS/core-web/pull/661/files, is that ok?

Reply:
Yes... But because I didn't merged I need it here too to work, I'll fix if there is any conflicts.